PlayerScripts/StarterCharacterScripts/Sound/LocalSound.client.lua

```diff
@@ -9,6 +9,10 @@
 	This script is optimized to reduce network traffic through minimizing the amount of property replication.
 ]]--
 
+--Roblox Services
+local Workspace=game:GetService("Workspace")
+local Players=game:GetService("Players")

```

Comment:
space needed between name and equals sign

Reply:
Yeah I'll fix that

Reply:
Just fixed it.